src/pybind/mgr/nfs/utils.py

```diff
@@ -67,7 +68,11 @@ def available_clusters(mgr: 'Module') -> List[str]:
     return value: ['vstart']
     '''
     # TODO check cephadm cluster list with rados pool conf objects
-    completion = mgr.describe_service(service_type='nfs')
+    try:
```

Comment:
Functionally it looks good to me! Can you attach a screenshot or a screencast of how Dashboard behaves now with changes? Thank you!

Reply:
Current behaviour on clicking 'NFS' tab:
![ceph-dashboard-nfs-current-behaviour](https://github.com/ceph/ceph/assets/14236359/71993fd6-dcac-4076-8a27-519310544a84)

New behaviour on clicking 'NFS' tab:
![ceph-dashboard-nfs-new-behaviour1](https://github.com/ceph/ceph/assets/14236359/a820bdd1-0e8f-4572-a5e9-709336f0b923)

On clicking 'create' (above):
![ceph-dashboard-nfs-new-behaviour2](https://github.com/ceph/ceph/assets/14236359/3019bc71-fd9f-4270-b8aa-dfff5104502d)

Then on clicking 'add a new NFS Service':
![ceph-dashboard-nfs-new-behaviour3](https://github.com/ceph/ceph/assets/14236359/552a5233-8b57-4330-bcfc-5eb8905229e1)


===

Current log excerpts from ceph-mgr:
```
2024-04-02T17:29:31.074+0000 7b26127fe640 -1 Remote method threw exception: Traceback (most recent call last): 
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/nfs/module.py", line 189, in cluster_ls
    return available_clusters(self)
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/nfs/utils.py", line 70, in available_clusters
    completion = mgr.describe_service(service_type='nfs')
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/orchestrator/_interface.py", line 1689, in inner
    completion = self._oremote(method_name, args, kwargs)
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/orchestrator/_interface.py", line 1756, in _oremote
    raise NoOrchestrator()
orchestrator._interface.NoOrchestrator: No orchestrator configured (try `ceph orch set backend`)

2024-04-02T17:29:31.078+0000 7b26127fe640  0 [dashboard ERROR controllers.nfs] Remote method threw exception: Traceback (most recent call last): 
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/nfs/module.py", line 189, in cluster_ls
    return available_clusters(self)
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/nfs/utils.py", line 70, in available_clusters
    completion = mgr.describe_service(service_type='nfs')
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/orchestrator/_interface.py", line 1689, in inner
    completion = self._oremote(method_name, args, kwargs)
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/orchestrator/_interface.py", line 1756, in _oremote
    raise NoOrchestrator()
orchestrator._interface.NoOrchestrator: No orchestrator configured (try `ceph orch set backend`)
Traceback (most recent call last): 
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/dashboard/controllers/nfs.py", line 273, in status 
    mgr.remote('nfs', 'cluster_ls')
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/mgr_module.py", line 2280, in remote 
    return self._ceph_dispatch_remote(module_name, method_name,
RuntimeError: Remote method threw exception: Traceback (most recent call last): 
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/nfs/module.py", line 189, in cluster_ls
    return available_clusters(self)
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/nfs/utils.py", line 70, in available_clusters
    completion = mgr.describe_service(service_type='nfs')
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/orchestrator/_interface.py", line 1689, in inner
    completion = self._oremote(method_name, args, kwargs)
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/orchestrator/_interface.py", line 1756, in _oremote
    raise NoOrchestrator()
orchestrator._interface.NoOrchestrator: No orchestrator configured (try `ceph orch set backend`)

2024-04-02T17:29:31.082+0000 7b26127fe640  0 [dashboard INFO request] [::ffff:127.0.0.1:37338] [GET] [200] [0.059s] [admin] [360.0B] /ui-api/nfs-ganesha/status
```

New log excerpts from ceph-mgr:
```
2024-04-02T17:37:40.937+0000 7dfd1a3fa640 20 mgr ~Gil Destroying new thread state 0x5c74f9c619e0
2024-04-02T17:37:40.945+0000 7dfd1a3fa640  0 [nfs ERROR nfs.utils] No orchestrator configured
Traceback (most recent call last):
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/nfs/utils.py", line 72, in available_clusters
    completion = mgr.describe_service(service_type='nfs')
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/orchestrator/_interface.py", line 1689, in inner 
    completion = self._oremote(method_name, args, kwargs)
  File "/home/ponnuvel/cases/ceph/src/pybind/mgr/orchestrator/_interface.py", line 1756, in _oremote
    raise NoOrchestrator()
orchestrator._interface.NoOrchestrator: No orchestrator configured (try `ceph orch set backend`)
```

Reply:
@avanthakkar I've uploaded screenshots as well the relevant log from ceph-mgr here. Please take a look. Thanks!